Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM ocaml/opam:debian-10-ocaml-4.11
+
+LABEL maintainer="gracewgao"
+
+SHELL ["/bin/bash", "-c"]
+
+# updates environment variables
+ENV PYRE_BINARY='/home/opam/pyre-check/source/_build/default/main.exe'
+ENV PYRE_TYPESHED='/home/opam/pyre-check/stubs/typeshed/typeshed-master'
+ENV PYTHONPATH='/home/opam/pyre-check:$PYTHONPATH'
+ENV PATH='/home/opam/.opam/4.10.2/bin:/home/opam/pyre-check/scripts:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:$PATH'
+
+RUN echo -e 'export PYRE_BINARY=/home/opam/pyre-check/source/_build/default/main.exe \n \
+    export PYRE_TYPESHED=/home/opam/pyre-check/stubs/typeshed/typeshed-master \n \
+    export PYTHONPATH=/home/opam/pyre-check:$PYTHONPATH \n \
+    export PATH=/home/opam/pyre-check/scripts:$PATH' >> ~/.bashrc
+
+# installs system dependencies
+RUN sudo apt update && sudo apt upgrade -y \
+    && sudo apt install python3 python3-pip pkg-config -y \
+    && sudo apt-get install libsqlite3-dev build-essential -y \ 
+    && opam switch create 4.10.2
+
+# downloads and installs pyre-check
+RUN git clone https://github.com/facebook/pyre-check \
+    && cd pyre-check \
+    && ./scripts/setup.sh --local
+    
+RUN cd pyre-check/source \
+    && make \
+    && make test
+
+# creates scripts for python alias and pyre-check
+RUN sudo touch /usr/bin/python \
```

Comment:
I think these four lines be done by creating a symlink? Eg. my `python` command currently is a symlink:
```
(venv) gbleaney-mbp:python_security gbleaney$ which python
/Users/gbleaney/python_security/venv/bin/python
(venv) gbleaney-mbp:python_security gbleaney$ ll /Users/gbleaney/python_security/venv/bin/python
lrwxr-xr-x  1 gbleaney  1876110778  7 Mar 31 23:43 /Users/gbleaney/python_security/venv/bin/python -> python3
```